### DISCUSSION TITLE
OsuLazer 2021 Not Working

### DISCUSSION THREAD

**Discussion by managerf3** [ppy/osu#15762] [Q&A]
Hello! So i was kinda bored of normal osu then i download and installed osulazer 2021 but when i opened it it didn't show me the osu screen just my background and my apps and when i tryed again i just saw a lil black bar on the top of my screen then i tried reopening it and that didn't work also so talk back if u know how to fix it

**Comment by nahkd123** [comment]
Please attach logs. See https://github.com/ppy/osu/discussions/15188

Also can you tell me a little bit about your system? (like operating system for example)

**Reply by managerf3** [reply]
Okay
Windows 8.1 Pro
IntelR CoreTM I5 CPU 760 2.80 GHz 
Ram: 4 GB
64 BIT

**Reply by nahkd123** [reply]
Uhhh, you forgot the logs...

See https://github.com/ppy/osu/discussions/15188 to know how to find logs and attach them here.

**Reply by nahkd123** [reply]
tl;dr: Check your GPU.

---

Ok so after a bit of thinking and researching, it sounds like you have some issues with your graphic card, or you don't have one. i5 760 itself doesn't have integrated GPU, so the only thing left here is whether or not you have dedicated/discrete GPU.

If you do have one, please make sure it supports OpenGL 4(?). If your GPU does not support, please make sure you're using the latest graphics driver. Sometimes upgrading driver allow you to run more OpenGL apps.

If you don't, well I guess you can't run osu!lazer, at least for now. There's currently an ongoing issue about adding "compatibility mode", which allow older hardware that doesn't support OpenGL to run osu!. Take a look at https://github.com/ppy/osu-framework/issues/4909.

However, **if you have any logs**, please attach them here. Logs allow everyone to help you troubleshoot your problem without guessing because of limited information (and most of the time it can help us solve problems faster). And again, if you need help on finding logs, see https://github.com/ppy/osu/discussions/15188
